VideoWeb/VideoWeb/ClientApp/src/app/services/api/consultation.service.ts

```diff
@@ -36,6 +37,8 @@ export class ConsultationService {
     consultationRequestee: Participant;
     consultationRequester: Participant;
 
+    consultationAcceptedBy: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
Please use the existing event service consultation message

Reply:
removed and use eventService. getConsultationMessage